### PR TITLE
Fixed the panic in subquery pullup.

### PR DESCRIPTION
**Pull request by guofengrichard**

### CODE REVIEW

File: src/backend/optimizer/plan/subselect.c [hunk 1]
```diff
@@ -1468,6 +1456,29 @@ convert_ANY_sublink_to_join(PlannerInfo *root, SubLink *sublink,
 	if (!bms_is_subset(upper_varnos, available_rels))
 		return NULL;
 
+	/*
+	 * GPDB_92_MERGE_FIXME:
```
Comment (danielgustafsson) [inline src/backend/optimizer/plan/subselect.c]: Does this need a FIXME comment once merged? Are we leaving any known open items from this PR?

**Review by danielgustafsson — COMMENTED** [review]
Seems reasonable.

File: src/backend/optimizer/plan/subselect.c (hunk 1 above)
Comment (guofengrichard) [reply src/backend/optimizer/plan/subselect.c]: I believe there is no known issues from this PR, but not with full confidence. Will remove the FIXEME once this PR is reviewed.

**Comment by danielgustafsson** [comment]
> On 28 Jun 2018, at 10:15, Richard Guo <notifications@github.com> wrote:
> 
> @guofengrichard commented on this pull request.
> 
> In src/backend/optimizer/plan/subselect.c <https://github.com/greenplum-db/gpdb-postgres-merge/pull/23#discussion_r198745893>:
> 
> > @@ -1468,6 +1456,29 @@ convert_ANY_sublink_to_join(PlannerInfo *root, SubLink *sublink,
>  	if (!bms_is_subset(upper_varnos, available_rels))
>  		return NULL;
>  
> +	/*
> +	 * GPDB_92_MERGE_FIXME:
> I believe there is no known issues from this PR, but not with full confidence. Will remove the FIXEME once this PR is reviewed.
> 
Figured as much, thanks for clarifying

**Pull request merged**